specification.md

```diff
@@ -1661,12 +1660,7 @@ transition: IssueRefundToCustomer
 
 Error definitions describe errors that can occur during workflow execution and how to handle them. 
 
-The `error`property defines the domain-specific name of the error. Users can also set the name to 
-`*` which is a wildcard specifying "all" errors, in the case where no other error definitions are defined,
-or "all other" errors if there are other errors defined within the same states `onErrors` definition.
-
-The `code` property can be used in addition to `name` to help runtimes resolve the defined 
-domain-specific error to the actual technical errors/exceptions that may happen during runtime execution.
+The `condition` property defines a query against error data returned from, say, a function call. Runtimes should detect data using technology-appropriate means (error codes, exception handling, etc.) and invoke error handling logic as appropriate.
```

Comment:
the way you structure your jq queries, again, its assumed that there is an "exceptions" array provided. I think this should again be mentioned somewhere